examples/environment-guide/backend-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1beta3
+kind: Service
+metadata:
+  creationTimestamp: null
+  name: backend-srv
+spec:
+  portalIP: ""
+  ports:
+  - name: ""
+    port: 5000
+    protocol: TCP
+    targetPort: 5000
```

Comment:
Remove targetPort if same as port.

You could remove protocol, even, but I prefer to leave that in for clarity.